keyboards/v60_type_r/keymaps/followingghosts/readme.md

```diff
@@ -0,0 +1,56 @@
+KB Paradise V60 Type R Customised Layout by FollowingGhosts
+======================
+
+This is my custom layout for the Polestar Type R
```

Comment:
You may want to say keymap instead of layout. In QMK, a layout is defined as the physical placement of keys on the board. A keymap is what each of those keys represent.

Reply:
Thanks, I've fixed that now